docs/plots/bokeh_backend.rst

```diff
@@ -0,0 +1,91 @@
+************************************
+The sherpa.plot.bokeh_backend module
+************************************
+
+The :term:`bokeh` plotting module is not integrated as tightly into
+IPython and Jupyter notebooks as :term:`matplotlib` is, so there is a little
+more setup required to make this work well.
+
```

Comment:
An issue I have with the above is that this assumes that this backend is only used with notebooks. This may in fact be the case, but if show we should probably be explicit about this in this paragraph.

Reply:
I can re-order. There is a section explicitly about the IPython terminal below that. My guess is that's is **mostly** going to be used with notebooks, and that's why I address that first, but I can re-order those sections.